.project-keeper.yml

```diff
@@ -6,7 +6,11 @@ sources:
       - integration_tests
       - udf_coverage
       - maven_central
+excludes:
+  - "W-PK-CORE-153: Project-keeper version 4.1.0 is outdated. Please update project-keeper to latest version 4.2.0."
```

Comment:
There are no customizations here. Is it possible to upgrade to 4.2.0 or is there another problem?

Reply:
I'll contact you on slack about this

Reply:
Updated PK